views/style.css

```diff
@@ -0,0 +1,38 @@
+body {
+  margin: 0px;
+  font-family: Helvetica, sans-serif, Arial;
+}
+
+#nav_bar {
+  width: 100%;
+  background-color: #333;
+  display: flex;
+}
+
+.navtab {
+  text-align: center;
+  padding: 8px 10px;
```

Comment:
I think you could make the navbar even thicker, especially if we're gonna put a logo up there. You could do that by adding vertical padding here or setting a height to `#nav_bar`.

Reply:
Done 22f2730
Looks like [this]( https://screenshot.googleplex.com/sFSCsOsiw2r) now